tests/upgrade_path/test_upgrade_path.py

```diff
@@ -86,6 +86,12 @@ def setup_upgrade_test(duthost, localhost, from_image, to_image, tbinfo,
     # Install target image
     logger.info("Upgrading to {}".format(to_image))
     target_version = install_sonic(duthost, to_image, tbinfo)
+    # Remove config_db.json before cold-reboot to target image
+    # to ensure the new config_db.json is generated from minigraph
+    if "cold" in reboot_type \
+            and duthost.shell("ls /host/old_config/minigraph.xml", module_ignore_errors=True)['rc'] == 0:
+        logger.info("Remove config_db.json before cold reboot to target image")
+        duthost.shell("rm -f /host/old_config/config_db.json", module_ignore_errors=True)
```

Comment:
Minor change, but use `duthost.command` instead of `duthost.shell` in these two places since the bash shell isn't required to run these commands (there's no glob expansion needing to be done).